opencti-platform/opencti-front/lang/front/fr.json

```
@@ -1656,6 +1659,7 @@
   "Max Confidence Level": "Niveau de confiance maximal",
   "Max confidence level": "Niveau de confiance maximal",
   "Max Confidence Level:": "Niveau de confiance maximum :",
+  "Max content level markings": "Marques de niveau de contenu maximum",
```

```suggestion
  "Max content level markings": "Marquage maximum du contenu",
```
it's all "marquage" in fench texts

---

indeed, fixed in several translations, thanks